### PR TITLE
Issue 53: Program overlay triangle

### PR DESCRIPTION
**Pull request by tmathern**
## Issue

Fix https://github.com/hlxsites/sgws-creative/issues/53

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/drafts/mathern/program-overlay/tab-view
- After: https://issue-53-program-overlay--sgws-creative--hlxsites.hlx.page/drafts/mathern/program-overlay/tab-view

### CODE REVIEW

File: blocks/tabview/tabview.css [hunk 1]
```diff
@@ -97,5 +97,18 @@
   text-transform: uppercase;
 }
 
+.clickable-program-overlay {
```
Comment (arumsey) [inline blocks/tabview/tabview.css:100]: Style names should indicate interactivity. You should also keep the class name namespaced within `.tabview` if that is where the elements will exists.

eg.

```
.tabview .program-overlay { ... }
```

File: blocks/tabview/tabview.js [hunk 2]
```diff
@@ -22,13 +22,32 @@ async function loadTabPanel(panel) {
...
+        panel.style.setProperty(`--${token}`, `${value}`);
+        tabButton?.style.setProperty(`--${token}`, `${value}`);
+      });
+    }
+  }
+
+  const programPath = panel.getAttribute('program-path');
+  if(programPath){
+    console.log("Fetching program fragment")
+    const programButton = document.createElement('div');
```
Comment (arumsey) [inline blocks/tabview/tabview.js:36]: Use `createTag` and then class name can be added on the same line.

File: blocks/tabview/tabview.js [hunk 3]
```diff
@@ -63,14 +82,18 @@ export default async function decorate(block) {
       });
     });
 
-    const anchor = tabContent.querySelector('a');
+    const anchors = tabContent.querySelectorAll('a');
+    const anchor = anchors[0];
```
Comment (arumsey) [inline blocks/tabview/tabview.js:86]: Use destructuring.

```
const [panelFragment, programFragment] = [...tabContent.querySelectorAll('a')];
```

File: blocks/tabview/tabview.js [hunk 4]
```diff
@@ -63,14 +82,18 @@ export default async function decorate(block) {
...
     const contentPath = anchor?.getAttribute('href');
     tabContent.textContent = '';
     tabContent.removeAttribute('class');
     tabContent.setAttribute('id', `tabpanel-${groupId}`);
     tabContent.setAttribute('role', 'tabpanel');
     tabContent.setAttribute('aria-labelledby', `tab=${groupId}`);
     tabContent.setAttribute('data-path', contentPath || '');
+    if(anchors.length > 1){
+      tabContent.setAttribute('program-path', anchors[1].getAttribute('href') || '');
```
Comment (arumsey) [inline blocks/tabview/tabview.js:95]: Could you still use `data-path` here? Is it still just a div that came from a fragment path.

Comment (tmathern) [reply blocks/tabview/tabview.js:127]: `data-path` is already used for the underlay.

**Pull request closed without merging**